### PR TITLE
1.19.4

### PR DESCRIPTION
**Pull request by Gerrygames**

### CODE REVIEW

**Review by Elikill58 — APPROVED** [review]

File: API/pom.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@
         <dependency>
             <groupId>org.inventivetalent</groupId>
             <artifactId>reflectionhelper</artifactId>
-            <version>1.18.9-SNAPSHOT</version>
+            <version>1.18.12-SNAPSHOT</version>
```
Comment (Elikill58) [inline API/pom.xml:39]: It seems only the 1.18.11-SNAPSHOT is available, not the 1.18.12-SNAPSHOT as showable [here](https://repo.inventivetalent.org/#browse/browse:public:org%2Finventivetalent%2Freflectionhelper%2Fmaven-metadata.xml)

Comment (Gerrygames) [reply API/pom.xml:39]: https://github.com/InventivetalentDev/ReflectionHelper/pull/59

Comment (Elikill58) [reply API/pom.xml:39]: > [InventivetalentDev/ReflectionHelper#59](https://github.com/InventivetalentDev/ReflectionHelper/pull/59)

Oh yes, thanks for fast answer and good luck for answer/merge :)

**Comment by srnyx** [comment]
Works for me, here's a built version for anyone else: https://media.srnyx.com/r/PacketListenerAPI_v3.9.10-SNAPSHOT.jar

**Review by srnyx — APPROVED** [review]

**Review by InventivetalentDev — APPROVED** [review]
didn't see this before, sorry!

**Pull request merged**